system/Build.jl

```diff
@@ -27,11 +27,12 @@ Oscar.system("precompile.jl")
 
 sysimage=joinpath(tmp, "Oscar.$(Libdl.dlext)")
 if !("JULIA_CPU_TARGET" in keys(ENV)) || (ENV["JULIA_CPU_TARGET"] == "")
-  PackageCompiler.create_sysimage([:Oscar], sysimage_path=sysimage, precompile_execution_file=CO)
+  println("Building system image for generic target. Use JULIA_CPU_TARGET to change.")
+  target = "generic;sandybridge,-xsaveopt,clone_all;haswell,-rdrnd,base(1)"
```

Comment:
That will only work on x86, though, not on e.g. my MacBook. How about this instead:
```suggestion
  target = PackageCompiler.default_app_cpu_target()
```